### PR TITLE
fix: add new fields

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to [APPS-1625](https://jira.library.ucla.edu/browse/APPS-1625) | [APPS-1626](https://jira.library.ucla.edu/browse/APPS-1626) | [APPS-1627](https://jira.library.ucla.edu/browse/APPS-1627) | [APPS-1628](https://jira.library.ucla.edu/browse/APPS-1628) | [PPS-1611](https://jira.library.ucla.edu/browse/APPS-1611)

### [Add alternative name (CJK) field to Staff Member detail page](https://jira.library.ucla.edu/browse/APPS-1625)

`alternativeName` field should go in a span within the `h1`

language code value in languageAltName field should be used in the span

e.g. `<h1 class="staffName">Yun-Hee Lee <span lang="ko">이윤희</span></h1>`

Screen Shot 2022-06-16 at 1 04 50 PM

---

### [Update Staff Member associations in Article detail page](https://jira.library.ucla.edu/browse/APPS-1626)

+ add associatedStaffMember as contact cards below article if value exists

Screen Shot 2022-06-16 at 3 55 18 PM

---

### [Add about field to Location detail page](https://jira.library.ucla.edu/browse/APPS-1627)

+ add about richTextSimplified field at the bottom of the location detail page

Screen Shot 2022-06-21 at 11 22 16 AM

---

## [Refactor/rename Research Guides to External Resources entry](https://jira.library.ucla.edu/browse/APPS-1628)

+ researchGuide becomes externalResource
+ researchGuideUrl becomes externalResourceUrl

---

## [Create Forms FPB](https://jira.library.ucla.edu/browse/APPS-1611)

- [x] The form field is added to all FB collection fragment query
- [x] Form is a flexible page block that is included in every flexible page block collection

### CODE REVIEW

File: pages/about/news/_slug.vue [hunk 1]
```diff
@@ -107,6 +107,176 @@ export default {
...
+// allFpb
+// bannerheader
+// date published
+
+// ServiceOrResources
+// share links & icons component
+// pipe instead of purple dot
+// fancy wayfinder divider
+
+<template lang="html">
```
Comment (pghorpade) [inline pages/about/news/_slug.vue:131]: template is repeating twice

File: pages/about/staff/_slug.vue [hunk 2]
```diff
@@ -4,10 +4,12 @@
         <!-- no search on this page -->
         <nav-breadcrumb :title="page.entry.title" />
 
+
         <block-staff-detail
             class="staff-detail-block"
             :image="parsedImage"
             :staff-name="parsedStaffName"
+            :alternative-name="page.entry.alternativeName"
```
Comment (pghorpade) [inline pages/about/staff/_slug.vue:12]: remove alternative-name and pass 2 props one for full text and one for language

**Review by pghorpade — CHANGES_REQUESTED** [review]

File: components/BannerHeader.vue [hunk 3]
```diff
@@ -292,6 +292,10 @@ export default {
             type: String,
             default: "",
         },
+        associatedStaffMember: {
```
[the remark this answers is not in the document]
Comment (jendiamond) [reply components/BannerHeader.vue:295]: I removed this.

File: components/BlockStaffDetail.vue [hunk 4]
```diff
@@ -172,6 +201,10 @@ export default {
             type: String,
             default: "",
         },
+        alternativeName: {
```
[the remark this answers is not in the document]
Comment (jendiamond) [reply components/BlockStaffDetail.vue:182]: I changed this to a String.

File: pages/about/news/_slug.vue (hunk 1 above)
Comment (jendiamond) [reply pages/about/news/_slug.vue]: fixed

File: pages/about/staff/_slug.vue (hunk 2 above)
Comment (jendiamond) [reply pages/about/staff/_slug.vue]:
```
            :alternative-full-name="parsedAlternativeFullName"
            :language="parsedLanguage"
```

**Comment by pghorpade** [comment]
Will merge this at noon today!

**Review by pghorpade — APPROVED** [review]
💯 good work.

**Pull request merged**